### PR TITLE
Add classes

### PR DESCRIPTION
**Pull request by eapenzacharias**
- Created class `Person`
- Created class `Student` and `Teacher` inherited from `Person`

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown')
+    super(age, name)
```
Comment (TedLivist) [inline student.rb]: - [x] Your `initialize` and `super` methods here are missing the parameter and arguments for the `parent_permission` attribute. Kindly add it.

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')
+    super(age, name)
```
Comment (TedLivist) [inline teacher.rb]: - [ ] Your `initialize` and `super` methods here are missing the parameter and arguments for the `parent_permission` attribute. Kindly add it.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @eapenzacharias,

Good job so far!
✔️ Good documentation
✔️ Classes are created in different files
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
#### _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eapenzacharias** [comment]
The pull request was merged by mistake. Please do review it I will work on any changes necessary
Thank you

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @eapenzacharias,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**